openwisp_radius/apps.py

```diff
@@ -14,12 +16,31 @@
 from .utils import load_model, update_user_related_records
 
 
-class OpenwispRadiusConfig(AppConfig):
+class OpenwispRadiusConfig(ApiAppConfig):
     name = 'openwisp_radius'
     label = 'openwisp_radius'
     verbose_name = 'Freeradius'
 
+    API_ENABLED = True
+    REST_FRAMEWORK_SETTINGS = {
+        'DEFAULT_THROTTLE_RATES': {
+            # None by default
+            'throttle_authorize': None,
+            'throttle_postauth': None,
+            'throttle_accounting': None,
+            # Strict throttling Policy
+            'throttle_obtain_auth': default_or_test('100/hour', None),
+            'throttle_validate_auth': default_or_test('100/hour', None),
+            # Lenient throttling Policy
+            'throttle_create_phone_token': default_or_test('250/hour', None),
+            'throttle_validate_phone_token': default_or_test('250/hour', None),
```

Comment:
same here, the problem is that openwisp-wifi-login-pages uses a request proxy so many users using the application will look like a single user.

Reply:
Okay, I have set
`throttle_authorize`, 
`throttle_postauth`, 
`throttle_accounting`

`throttle_obtain_auth`, 
`throttle_validate_auth`, 

`throttle_create_phone_token`,
`throttle_validate_phone_token`

to `None` and the rest of them as `400/hour` as default. Is It okay?

Reply:
There's no need to prepend `throttle_` right? Can we just have the name of the view?

Reply:
Sure, @nemesisdesign Done :+1: